scripts/allocate-compute.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+import argparse, sys, time
+
+import common, connection
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Allocate storage and assign it to a coumpte node',
+        epilog = 'Must executed on Rabbit with nnf-ec running')
+    
+    parser.add_argument('--node', nargs='+', choices=range(0,16), default=None, help='the node index to attach to')
+    parser.add_argument('--size', type=str, action=common.ByteSizeAction, default=common.ByteSizeStringToIntegerBytes('500GB'), help='storage pool size (default 500GB)')
+
+    connection.addServerArguments(parser)
+
+    args = parser.parse_args()
+
+    if args.node is None:
+        args.node = range(0, 16)
+
+    conn = connection.connect(args, '/redfish/v1/StorageServices/NNF')
+
+    for node in args.node:
+        print(f"Starting Node '{node}' Sequence...")
```

Comment:
For the prints, you're using double quotes in some places and single quotes elsewhere.

Reply:
Double only when string uses single quotes